_posts/2018-08-15-Episode-0-Bokeh.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: Open Source Directions Episode 0: Bokeh
+---```
+
+### Introductions
+
+This is the webinar that informs the community about what is going on with Open Source Python projects.  I am Anthony Scopatz and today the project being highlighted is Bokeh.
+
+Tony Fast joins us as a community organizer and open source consultant.
+
+Bryan Vandavin is our second guest, and he works for Anaconda and was a developer of conda, but has since begun working on Bokeh.
```

Comment:
I think this should be written less like I am speaking it and more like.

"Our host is Anthony, Our guests are..."